backend/typescript/server.ts

```diff
@@ -61,7 +65,45 @@ app.get("/delete-user-by-email/:email", async (req, res) => {
   await userService.deleteUserByEmail(req.params.email);
   res.status(204).send();
 });
-/** ************************************************************************ */
+
+const authService: IAuthService = new AuthService();
+
+app.get("/generate-token/:email/:password", async (req, res) => {
```

Comment:
Can we put these routes in a different file and use the express router to keep the server file clean?

Reply:
Will do as part of my next task! These are temporary, just to enable testing